test/integration/script-loader/base/pages/page1.js

```diff
@@ -3,14 +3,19 @@ import Script from 'next/script'
 const Page = () => {
   return (
     <div class="container">
-      <Script
-        id="scriptBeforeInteractive"
-        src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.20/lodash.min.js?a=scriptBeforeInteractive"
-        strategy="beforeInteractive"
```

Comment:
Can we keep some tests with `beforeInteractive` outside of `_document` to ensure we aren't breaking with a comment saying it can be removed in a major?

Reply:
Added a new test to capture this